ui/components/__tests__/Breadcrumbs.test.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { createMockClient, withContext, withTheme } from "../../lib/test-utils";
+import Breadcrumbs from "../Breadcrumbs";
+
+describe("snapshots", () => {
+  it("renders", () => {
+    const tree = renderer
```

Comment:
:+1: 

Can we get another snapshot case that shows a child route being rendered?